meta-balena-common/recipes-core/balena-rollback/files/rollback-clear-bootcount

```diff
@@ -24,7 +24,9 @@ if [ -f "/mnt/state/rollback-altboot-breadcrumb" ]; then
 	# Clear grub bootcount
 	grub_env=$(find "${BALENA_BOOT_MOUNTPOINT}" -name grubenv)
 	if [ -f "${grub_env}" ]; then
-		sed -i "s#bootcount=.*#bootcount=0 #g" "${grub_env}" || true
+		tmpfile=$(mktemp)
+		"${CAT}" "${grub_env}" | sed "s#bootcount=.*#bootcount=0 #g" > "${tmpfile}" || true
+		"${MV}" "${tmpfile}" "${grub_env}"
```

Comment:
We can also avoid using a temporary file by piping the result back to `fatrw write` like:
```
fatrw read test.txt | sed 's/hi/HELLO/' | fatrw write test.txt
```

Reply:
thanks @majorz makes sense to avoid a tmp file.